exchanges/bitstamp/bitstamp_test.go

```diff
@@ -198,57 +185,71 @@ func TestGetAccountTradingFees(t *testing.T) {
 func TestGetTicker(t *testing.T) {
 	t.Parallel()
 
-	_, err := b.GetTicker(context.Background(),
+	ticker, err := b.GetTicker(context.Background(),
 		currency.BTC.String()+currency.USD.String(), false)
-	if err != nil {
-		t.Error("GetTicker() error", err)
-	}
+	assert.NoError(t, err, "GetTicker should not error")
+	assert.Positive(t, ticker.Ask, "Ask should be positive")
+	assert.Positive(t, ticker.Bid, "Bid should be positive")
+	assert.Positive(t, ticker.High, "High should be positive")
+	assert.Positive(t, ticker.Low, "Low should be positive")
+	assert.Positive(t, ticker.Last, "Last should be positive")
+	assert.Positive(t, ticker.Open, "Open should be positive")
+	assert.Positive(t, ticker.Volume, "Volume should be positive")
+	assert.Positive(t, ticker.Vwap, "Vwap should be positive")
+	assert.NotEmpty(t, ticker.Timestamp, "Timestamp should not be empty")
+	assert.Contains(t, []order.Side{order.Buy, order.Sell}, ticker.Side.Side(), "Side should be either Buy or Sell")
 }
 
 func TestGetOrderbook(t *testing.T) {
 	t.Parallel()
-	_, err := b.GetOrderbook(context.Background(),
-		currency.BTC.String()+currency.USD.String())
-	if err != nil {
-		t.Error("GetOrderbook() error", err)
+	ob, err := b.GetOrderbook(context.Background(), currency.BTC.String()+currency.USD.String())
+	assert.NoError(t, err, "GetOrderbook should not error")
+	assert.NotEmpty(t, ob.Timestamp, "Timestamp should not be empty")
+	for i, o := range [][]OrderbookBase{ob.Asks, ob.Bids} {
+		s := []string{"Ask", "Bid"}[i]
+		if assert.NotEmptyf(t, o, "Should have items in %ss", s) {
+			a := o[0]
+			assert.Positivef(t, a.Price, "%ss Price should be positive", s)
+			assert.Positivef(t, a.Amount, "%s Amount should be positive", s)
```

Comment:
```suggestion
			assert.Positivef(t, a.Amount, "%ss Amount should be positive", s)
```

To keep it consistent with usage of plurals like the others

Reply:
Amended